deployment/helm/node-feature-discovery/templates/master.yaml

```diff
@@ -65,6 +65,7 @@ spec:
           ports:
           - containerPort: 8080
             name: grpc
+            hostPort: {{ .Values.master.port }}
```

Comment:
I don't think it is good idea to use `hostPort`  as a default option to deploy `nfd-master` it will use/reserve this port on the k8s worker node. If we just want to make it configurable on which port `nfd-master` is listening then something like this would be enough:
```yaml
              command:
              - "/usr/bin/grpc_health_probe"
              - "-addr=:{{ .Values.master.port | default "8080" }}"
              
              ...
          ports:
          - containerPort: {{ .Values.master.port | default "8080" }}
            name: grpc
```
reference to best practises:
https://kubernetes.io/docs/concepts/configuration/overview/#services

Reply:
Thanks @PiotrProkop, you're absolutely right 🙄 My feedback was ill-advised.

@AhmedGrati forget about my thoughtless comments, listen to @PiotrProkop

Reply:
Okay, I will update it ASAP.